tests/integration_tests/base_api_tests.py

```diff
@@ -288,6 +288,30 @@ def test_get_filter_related_owners(self):
         # TODO Check me
         assert expected_results == sorted_results
 
+    def test_get_base_filter_related_owners(self):
+        """
+        API: Test get base filter related owners
+        """
+        self.login(username="admin")
+        uri = f"api/v1/{self.resource_name}/related/owners"
+        self.app.config["EXCLUDE_USER_USERNAMES"] = ["gamma"]
```

Comment:
Not something that needs to be tackled here, but wouldn't it be nice with a `with_config` decorator, similar to the `with_feature_flags` one?

Reply:
nice suggestion, implemented a `with_config`